src/frontend/src/lib/components/ui/ButtonControl.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	export let ariaLabel: string;
+	export let styleClass: string | undefined = undefined;
+</script>
+
+<button
+	aria-label={ariaLabel}
+	class={`${styleClass} h-7 w-7 rounded-md bg-white p-1 shadow transition-colors duration-700 hover:text-blue-ribbon`}
```

Comment:
`styleClass ?? ""` otherwise are we maybe ending with a class name "undefined" in the DOM?

Reply:
True, I changed the default value for `styleClass` to `''` instead of `undefined`